HW1/P2/P2.txt

```diff
@@ -1 +1,10 @@
 # Your discussion here
+For this problem 2, I generated bar plots with partitions for X axis and corresonding efforts for Y axis instead of histograms mentioned in the HW1 manual so that I can easily see how much each effort is distributed roughly the same on each partition. 
```

Comment:
I am not understanding what you mean. From what I see, you did plot the histograms, but the difference is what you are plotting in the y-axis?